package.json

```diff
@@ -107,13 +107,13 @@
     "nps-utils": "^1.5.0",
     "prettier": "^2.6.2",
     "prop-types": "^15.8.1",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
+    "react": "^18.1.0",
+    "react-dom": "^18.1.0",
     "react-hot-loader": "v2.0.0-alpha-4",
-    "react-test-renderer": "^16.13.1",
+    "react-test-renderer": "^18.1.0",
     "rimraf": "^3.0.2",
     "seedrandom": "^2.4.3",
-    "storybook-addon-performance": "^0.10.0",
+    "styled-components": "^5.3.5",
```

Comment:
Why was this added?

Reply:
We are using `styled-components` in some of our storybook files. I'm not sure why the imports were all working before!